cartography/intel/aws/ec2.py

```diff
@@ -255,6 +255,7 @@ def load_ec2_instances(neo4j_session, data, region, current_aws_account_id, aws_
                 ImageId=instance.get("ImageId", ""),
                 SubnetId=instance.get("SubnetId", ""),
                 InstanceType=instance.get("InstanceType", ""),
+                IamInstanceProfile=instance.get("IamInstanceProfile", {}).get("Arn", ""),
```

Comment:
Would prefer to leave a field blank than add an empty string.

Change this to 
```
IamInstanceProfile=instance.get("IamInstanceProfile", {}).get("Arn")
```

and I think it should be good.  Once you do that, we'll test internally and then get back to you :)

Reply:
Done. I appreciate the quick response!